### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/apngasm/commit/df3fce127109cfe29b68a049e23f769aedc2639f#commitcomment-79892982

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/apngasm/commit/df542396ddc91a864083950dca1c51435dc9562b

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: docs/building/win/instructions.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apngasm Build Instructions for Windows
 --------------------------------------
 This guide was written for Windows 8 using Visual Studio 2013.  
-Building on Windows is significantly more complex than Linux or OSX. Due to this an other factors this guide will likely not be actively maintained. If you find part of this guide needs revision please do not report it as an issue. Instead; fork the repository on github, make your revisions, and submit a pull request.
+Building on Windows is significantly more complex than Linux or OSX. Due to this and other factors this guide will likely not be actively maintained. If you find part of this guide needs revision please do not report it as an issue. Instead; fork the repository on github, make your revisions, and submit a pull request.
```
Comment (jsoref) [inline docs/building/win/instructions.md:4]: This is an atypical correction (my default is `an other` -> `another`)

File: docs/building/win/instructions.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Build zlib and libpng
 Build boost
 ===========
 1. Decompress the boost sources.
-2. Open a command prompt and navigate to the directory where you decompressed bost.
+2. Open a command prompt and navigate to the directory where you decompressed boost.
```
Comment (jsoref) [inline docs/building/win/instructions.md:32]: Brand

File: docs/html/search/search.js [hunk 3]
```diff
@@ -543,7 +543,7 @@ function SearchResults(name)
       return true;
     }
 
-    // return the first item with index index or higher that is visible
+    // return the first visible item with index greater than or equal to index
```
Comment (jsoref) [inline docs/html/search/search.js:546]: This change is because `index index` is really hard to parse...

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine.

File: docs/building/win/instructions.md (hunk 1 above)
Comment (pfennig59) [reply docs/building/win/instructions.md:4]: Actually, the correction is correct, because its not correcting another but and. Another makes no sense here.
